src/Library/demos/Spinner/main.blp

```diff
@@ -0,0 +1,46 @@
+using Gtk 4.0;
+using Adw 1;
+
+Adw.StatusPage {
+  title: "Spinner";
+  description: _("Display loading state");
+
+  Box {
+    orientation: vertical;
+    halign: center;
+
+    Box {
+      orientation: vertical;
+      halign: center;
+
+       Spinner spinner {
+          spinning: true;
+          margin-bottom: 30;
+        }
+
+      Button  btn {
+        icon-name: "media-playback-stop";
```

Comment:
hmm - I found it difficult to recognize what it is - what do you think of using `media-playback-pause` instead?

Reply:
Yep would look even better I'd update it